hooks/conf_regen/25-dovecot

```diff
@@ -53,6 +53,8 @@ do_post_regen() {
     chown root:mail /var/mail
     chmod 1775 /var/mail
 
+    python3 -c 'from yunohost.app import regen_mail_app_user_config_for_dovecot_and_postfix as r; r(only="dovecot")'
```

Comment:
Note that we have these weird lines in dovecot and postfix regenconf because ... I need a regenconf-like mechanism but also wanted to trigger the regeconf from the resource reconciliation python snippet ... And the code that generate these files was easier to write in Python ...